### PR TITLE
Added support to Symfony 5

### PR DESCRIPTION
**Pull request by gnugat**
Added support to:

* Symfony YAML 5
* Symfony Filesystem 5 (dev)
* Symfony PHPUnit bridge 5 (dev)

### CODE REVIEW

**Comment by gnugat** [comment]
This should address #131.
Not sure why #128 got closed, and from the look of it #130 isn't providing Symfony 5 support.

**Comment by gnugat** [comment]
Tests fail due to:
* PHP 7.2: use of PHPUnit 8 (tests pass with PHPUnit 7)
* PHP 5.5, 5.4: no longer supported by phpenv (used in Travis)

I recommend dropping older PHP versions, and perhaps upgrading tests to PHPUnit 8 (see #129 to solve both), and even maybe dropping support to older symfony versions (see #130).

**Comment by stof** [comment]
>     * PHP 5.5, 5.4: no longer supported by phpenv (used in Travis)

Actually, they can be supported by forcing `dist: trusty` (which was the default previously)

**Comment by gnugat** [comment]
Good news, I managed to fix the PHPUnit BC break (Symfony PHPUnit Bridge allows a `SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT=1` environment variable to fix that issue).
Bad news, the `dist: trusty` doesn't seem to do the trick, although I'm not quite sure why...

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ env:
     global:
         - SYMFONY_PHPUNIT_DIR=$HOME/.phpunit-bridge
         - SYMFONY_PHPUNIT_REMOVE='' # don't remove prophecy
+        - SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT=1
 
 php: [7.2, 7.1, 7.0, 5.6, 5.5, 5.4]
```
Comment (derrabus) [inline .travis.yml]:
```suggestion
 php: [7.2, 7.1, 7.0, 5.6]
```

You're adding 5.4 and 5.5 runs via the matrix, so you don't need them here anymore.

Comment (gnugat) [reply .travis.yml]: thanks, I'll add your suggestion to the PR

Comment (gnugat) [reply .travis.yml]: Looks like travis is no longer able to parse the config file

Comment (derrabus) [reply .travis.yml]: Yes, there‘s now an extra whitespace before the `php` that does not belong there.

Comment (gnugat) [reply .travis.yml]: gotcha, ta

**Comment by gnugat** [comment]
Bruce Willis threatening: "Green?", Chris Tucker answering "Supergreen" From the movie the 5th Element

**Comment by adrienfr** [comment]
Hi, do you know when a release can be expected to enjoy the SF5 compatibility ? Thanks!

**Pull request merged**